docs/explanations/audit-scanner.md

```diff
@@ -0,0 +1,214 @@
+---
+sidebar_label: "Audit Scanner"
+title: "Kubewarden Audit Scanner"
+---
+
+# Kubewarden Audit Scanner
+
+Starting from version `v1.7.0`, Kubewarden introduces a new component called
+the Audit Scanner. This scanner evaluates whether the resources running in the
+cluster comply with the deployed policies. It is particularly useful when
+policies are deployed after the resources. The Audit Scanner allows operators
+to detect resources deployed before the policies that violate the validations
+performed by the policies.
+
+See [here](../howtos/audit-scanner) how to enable Audit Scanner
+
+### Policy Reports
+
+The results of the policy reports generated by the Audit Scanner are stored in
+Kubernetes using the Policy Report CRDs. If you want to visualize and monitor
+these reports, you can use the Policy Reporter tool available at [policy-reporter](https://github.com/kyverno/policy-reporter).
+
+The Policy Reporter tool provides an optional UI to view and analyze the policy
+reports, offering enhanced monitoring and observability capabilities.
+
+When using the Kubewarden Audit Scanner, the results of the policy scans are
+stored in Policy Report Custom Resource Definitions (CRDs). Here are some
+examples of policy reports and how they are structured:
+
+To illustrate the usage of the audit scanner in Kubewarden, let's consider the
+following scenario:
+
+Assume you have deployed a set of policies to enforce specific rules and
```

Comment:
I think these paragraphs shouldn't be under `### Policy Reports` but merged outside, as they aren't explaining thing about PolicyReports